tests/test_sqlcompletion.py

```diff
@@ -99,14 +99,27 @@ def test_suggests_tables_views_and_schemas(expression):
 
 @pytest.mark.parametrize('expression', [
     'SELECT * FROM ',
+])
+def test_suggest_tables_views_schemas_and_functions(expression):
+    suggestions = suggest_type(expression, expression)
+    assert set(suggestions) == set([
+        Table(schema=None),
+        View(schema=None),
+        Function(schema=None, filter='for_from_clause'),
+        Schema()
+    ])
+
+
+@pytest.mark.parametrize('expression', [
     'SELECT * FROM foo JOIN ',
```

Comment:
We probably need another test for a second level join, e.g. `SELECT * FROM foo JOIN bar ON foo.id = bar.id JOIN `